client/asset/btc/electrum_client.go

```diff
@@ -1254,6 +1255,36 @@ func (ew *electrumWallet) sendToAddress(address string, value, feeRate uint64, s
 	return chainhash.NewHashFromStr(txid)
 }
 
+// part of btc.Wallet interface.
+func (ew *electrumWallet) estimateSendTxFee(tx *wire.MsgTx, feeRate uint64, subtract bool) (fee uint64, err error) {
+	txBytes, err := ew.serializeTx(tx)
+	if err != nil {
+		return 0, fmt.Errorf("tx serialization error: %w", err)
+	}
+
+	// 1e-5 = 1e-8 for satoshis * 1000 for kB.
+	feeRateOption := float64(feeRate) / 1e5
+	options := &btcjson.FundRawTransactionOpts{
+		FeeRate:    &feeRateOption,
+		ChangeType: &btcjson.ChangeTypeBech32,
+	}
+	if subtract {
+		options.SubtractFeeFromOutputs = []int{0}
+	}
+	if !ew.segwit {
+		options.ChangeType = &btcjson.ChangeTypeLegacy
+	}
```

Comment:
Electrum has no relation to the btcjson types, which only pertain to the **bitcoin core clone** RPC wallets.  Electrum's RPC is completely different.  I'll suggest a diff.